src/pages/Signin/Kakao.js

```diff
@@ -0,0 +1,32 @@
+import React, { useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+export default function Kakao() {
+  //code를 추출하기 위한 과정
+  const href = window.location.href;
+  let params = new URL(document.URL).searchParams;
+  let code = params.get('code');
+
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    fetch(`http://10.58.52.162:3000/user/signin?code=${code}`, {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json;charset=utf-8',
+      },
+    })
```

Comment:
```suggestion
    fetch(`http://10.58.52.162:3000/user/signin?code=${code}`)
```

GET 요청일때는 API 주소만 작성해도 됩니다!